app/templates/home.html

```diff
@@ -174,7 +180,33 @@ <h2 class="fw-bold text-success" id="thank-you-header">🎉 Thank You for Attend
         from { opacity: 0; transform: scale(0.9); }
         to { opacity: 1; transform: scale(1); }
     }
+
+    #category-container {
+    -ms-overflow-style: none; /* Hide scrollbar for IE and Edge */
+    scrollbar-width: none; /* Hide scrollbar for Firefox */
+    }
+
+    #category-container::-webkit-scrollbar {
+        display: none; /* Hide scrollbar for Chrome, Safari, and Opera */
+    }
 </style>
 
+<script>
+    document.addEventListener("DOMContentLoaded", function () {
+        let container = document.getElementById("category-container");
+
+        function autoScroll() {
+            if (container.scrollLeft + container.clientWidth >= container.scrollWidth) {
+                container.scrollLeft = 0; // Reset scroll when reaching the end
+            } else {
+                container.scrollLeft += 1; // Adjust speed if needed
+            }
+        }
+
+        setInterval(autoScroll, 50); // Adjust speed by changing interval
```

Comment:
```suggestion
        setInterval(autoScroll, 16); // Adjust speed by changing interval
```
Reducing the interval to match 60fps would make the scrolling smoother, although it would also speed it up slightly. Up to you whether you want to do that. `25` might also be a decent option.